### PR TITLE
Use alpine image to save resources

### PR DESCRIPTION
**Pull request by EHER**
Small images may save some re$ource$ :)

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,14 +1,5 @@
-FROM node:stretch-slim
-
-RUN apt update && apt install -y git openssh-server ca-certificates openssl jq gettext xmlstarlet build-essential
-
-RUN npm install sfdx-cli prettier prettier-plugin-apex --global
```
Comment (airtonzanon) [inline Dockerfile:5]: Won't we lose the prettier here?

File: Dockerfile [hunk 2]
```diff
@@ -1,14 +1,5 @@
...
+FROM node:alpine
+RUN npm install --global sfdx-cli
 RUN sfdx --version
-RUN sfdx plugins --core
-
-# Work-around to make java installation work
-# as per https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=863199
-RUN mkdir -p /usr/share/man/man1
```
Comment (airtonzanon) [inline Dockerfile:11]: Will we have this in another image?

File: Dockerfile [hunk 3]
```diff
@@ -1,14 +1,5 @@
-FROM node:stretch-slim
-
-RUN apt update && apt install -y git openssh-server ca-certificates openssl jq gettext xmlstarlet build-essential
-
-RUN npm install sfdx-cli prettier prettier-plugin-apex --global
+FROM node:alpine
```
Comment (airtonzanon) [inline Dockerfile:1]:
```suggestion
FROM codelicia/apex-prettier:latest
```

**Review by airtonzanon — APPROVED** [review]

File: Dockerfile [hunk 4]
```diff
@@ -1,14 +1,5 @@
-FROM node:stretch-slim
-
-RUN apt update && apt install -y git openssh-server ca-certificates openssl jq gettext xmlstarlet build-essential
-
-RUN npm install sfdx-cli prettier prettier-plugin-apex --global
+FROM codelicia/apex-prettier:latest
+RUN npm install --global sfdx-cli
 RUN sfdx --version
```
Comment (lcobucci) [inline Dockerfile]: I'd suggest merging this command with the previous `RUN`. Having an extra cache layer with this seems unnecessary to me 😬

Comment (lcobucci) [reply Dockerfile]: (not to mention you will save even more re$ource$)

**Comment by EHER** [comment]
@airtonzanon `codelicia/apex-prettier:latest` was breaking the build...

File: Dockerfile (hunk 4 above)
Comment (EHER) [reply Dockerfile]: That is a good point! --> .

Thank you! ❤️

**Review by airtonzanon — APPROVED** [review]

**Comment by airtonzanon** [comment]
That's cool for me, but perhaps, if we have someone else than us using this image, they'll lose prettier

**Pull request merged**